src/_guides/language/sound-faq.md

```diff
@@ -0,0 +1,189 @@
+
+### Is Dart still optionally typed?
+Strong mode is optional, so you can continue to write Dart code as before.
+However, there are many benefits to sound code, such as finding bugs
+at compile time rather than runtime. For more information, see
+[The benefits of soundness](/guides/language/sound-dart#the-benefits-of-soundness).
+
+### What is the difference between strong mode and checked mode?
+
+Checked mode performs limited type checking but doesn’t guarantee that
+an expression evaluates to a specific type at runtime.
+Strong mode performs stronger type checking. For more information, see
+[Strong mode vs. checked mode](/guides/language/sound-dart#strong-mode-vs-checked-mode).
+
+### How much work is it to switch to strong mode?
+
+Developers at Google have migrated tens of thousands of lines of
+Dart to strong mode. Many of the changes involved adding annotations
+to Maps and Lists. Often, making code strong mode clean is a matter
+of making good use of generic methods. Finally, code that uses dynamic
+types to hold other types must be cleaned up.
+
+### Can some of my code in my app/library be strong mode, and other code be not-strong-mode?
+
+You can exclude files from static analysis. For more information, see
+[Excluding files](/guides/language/analysis-options#excluding-files).
+However, if you are using DDC, the code must be entirely strong mode clean.
+
+### Are the Dart core libraries compliant with strong mode?
+
+Yes, as of version 1.21.
+
+### Are the packages produced by the Dart team compliant with strong mode?
+
+Many packages have been updated to be strong mode compliant and more
+are in the works. Please file an issue against a package that you use
+but isn’t yet strong mode compliant.
+
+### What happens if I write strong mode code, but I use packages that are not compliant with strong mode?
+
+Other packages are excluded from static analysis by default. You have
+to specify `--show-package-warnings` to see errors in other packages.
+As for executing code, the dart2js and Dart VM tools don’t currently
+support strong mode so, yes, you can mix and match.
+DDC requires strong mode compliance and won’t compile code that is unsound.
+
+### Is strong mode Dart's new type system?
+
+Strong mode is part of the implementation of Dart’s new type system,
+which is currently under development. For more information, see
+[What constitutes strong mode](/guides/language/sound-dart#what-constitutes-strong-mode).
+
+### Is Dart going to turn into a verbose, cumbersome language where I have to put types everywhere?
+
+No! Strong mode Dart continues to allow `var` and `dynamic` with type
+inference in many places. Whenever possible, the analyzer infers a type.
+When the analyzer can’t infer a type,
+it generates an error and the type must be explicitly defined.
+
+### I liked Dart before. Can I still write Dart code the way I always did?
+
+Yes, you can write Dart code as before because strong mode is optional.
```

Comment:
"Even with strong mode, though, Dart stays the same terse and familiar language you liked."